### PR TITLE
Rename Engineering team to Site Reliability Engineering team

### PR DESCRIPTION
**Pull request by ohbarye**
@quipper/sre Please check and merge, thanks.

### CODE REVIEW

File: company-profile-en.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
 - The team has grown to over 500 employees, including our largest sales force of 300 staff across Indonesia in 12 local branches.
 - The product team consists of c. 50 members, which more than half are web engineers. (as of Sep 2016)
 
-|| Web | iOS/Android | Engineering | Design | Total |
+|| Web | iOS/Android | Site Reliability Engineering | Design | Total |
```
Comment (hakobera) [inline company-profile-en.md]: I think just `SRE` is enough.

File: company-profile-ja.md [hunk 2]
```diff
@@ -22,7 +22,7 @@
 - プロダクト開発を行うチームは全体で 50 名弱です。半数以上がエンジニアで、エンジニアの半数以上が web エンジニア（サーバサイド）です
 - 参考：2016年9月末時点
 
-|| Web | iOS/Android | Engineering | Design | Total |
+|| Web | iOS/Android | Site Reliability Engineering | Design | Total |
```
Comment (hakobera) [inline company-profile-ja.md]: Same as english version

**Comment by ohbarye** [comment]
@hakobera 
Thanks, I fixed them.

**Comment by ohbarye** [comment]
Thank you @hakobera @banyan

**Pull request merged**